### DISCUSSION TITLE
Replace Order with $0 amount

### DISCUSSION THREAD

**Discussion by amitkathpal123** [SalesforceCommerceCloud/sfcc-ci#445] [Team Posts]
Hi All,

Can you please provide your thoughts on achieving replace order with $0 amount.

We are allowing customer service to place replaced orders (in case order is misplaced or for some other reasons) with $0 amount. 

1. Is it possible to place $0 order in SFCC (no shipping fee as well)?
2. I looked at API's and found below. Is that the one to use or there is more?
static createBasketFromOrder(order : Order) : Basket

Thanks,
Amit

**Comment by astound-scollins** [comment]
Hi @amitkathpal123, 

Yes, it is possible to create a 0 dollar order in SFCC, and there are a couple of ways that it could be done. 

Generally the main considerations for this types of orders is that there will generally not be a billing address and a payment instrument associated with the orders, so you need to ensure that all of the down-stream systems are capable of still processing the order without that information, or you will need to dummy / copy that information from the previous order. 

As for getting to the $0.00 order, you can either use manual price adjustments to set the line item costs for all of the items on the order to $0.00, as well as the shipping cost to $0.00, or you can use customer service promotion / coupon codes or source codes to do the same thing. By going the customer service promotion route you will have that promotion code available as a reporting tool as well to be able to see how often free orders are going out from customer service.

**Comment by amitkathpal123** [comment]
Thanks for responding.

I am trying the second option with coupon code but anytime total order value goes 0, it does not work. It should be minimum of 1cent. Do you have it working on any sandbox/storefront, i could check.

**Comment by pravngaur** [comment]
Hey Amit,
I remember i used to work in past, don't know if anything been changed
recently. Will give a try today.
